tests/lib/cli-engine/cli-engine.js

```diff
@@ -5184,21 +5184,21 @@ describe("CLIEngine", () => {
         });
 
         it("should expose the list of plugin rules", () => {
-            const engine = new CLIEngine({ plugins: ["n"] });
+            const engine = new CLIEngine({ plugins: ["internal-rules"] });
```

Comment:
Just to clarify -- was this done because `eslint-plugin-n` is no longer installed in a location where the test can read it?

Reply:
it's because `eslint-plugin-n` v16 has dropped node.js v12 supports, while eslint still supports.

https://github.com/eslint-community/eslint-plugin-n/releases/tag/16.0.0